src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,29 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo" key={post.id}>
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
theoretically, user post.user might be undefined (getUser can return undefined). Please, handle this case (use [conditional rendering](https://react.dev/learn/conditional-rendering#logical-and-operator-))